website/src/pages/batch-changes.tsx

```diff
@@ -0,0 +1,265 @@
+            <h1 className="mb-3">Change code everywhere with a single declarative file</h1>
+            <div className="row justify-content-center">
+                <p className="col-md-8">
+                    Batch Changes gives you a declarative structure for finding and modifying code across all of your
+                    repositories. With a simple UI, it is easy to track and manage all of your changesets through checks
+                    and code reviews until each change is merged.
+                </p>
+            </div>
+            <div className="row">
+                <div className="col-md-6 mt-4">
+                    <figure>
+                        <img
+                            src="/batch-changes/asking_developers_to_update_repos.svg"
+                            alt="Graphic of many developers individually updating repositories, while some are not updating their repositories."
+                        />
+                        <figcaption className="mt-5">
+                            Asking <strong>developers</strong> to update repositories
+                        </figcaption>
+                    </figure>
+                </div>
+                <div className="col-md-6 mt-4">
+                    <figure>
+                        <img
+                            src="/batch-changes/using_batch_changes_to_update_repos.svg"
+                            alt="Graphic of a single developer updating a lot of repositories."
+                        />
+                        <figcaption className="mt-5">
+                            Using <strong className="text-purple">Batch Changes</strong> to update repositories
+                        </figcaption>
+                    </figure>
+                </div>
+            </div>
+        </ContentSection>
+
+        {/* Architecture */}
+        <div className="bg-gradient-green-blue py-4 py-md-7">
+            <ContentSection>
+                <div className="row">
+                    <div className="col-lg-5">
+                        <h1 className="mb-3">Search, define, execute, and track code changes</h1>
+                        <ul className="list-spaced">
+                            <li>
+                                Find all occurrences of code to change with Sourcegraph{' '}
+                                <a href="http://about.sourcegraph.com/">universal code search</a>
+                            </li>
+                            <li>Programmatically define changes by creating a declarative specification file</li>
+                            <li>Execute specifications via a lightweight CLI</li>
+                            <li>
+                                Track changeset lifecycle status across multiple code hosts via the Sourcegraph UI
+                            </li>
+                        </ul>
+                    </div>
+                    <div className="col-lg-7">
+                        <Video name="how-it-works" />
```

Comment:
On mobile, the animation doesn't run (at least on Netlify). When the page is live, will the animation play?

Reply:
It plays for me on iOS on Netlify. What did you test this with?